### ISSUE TITLE
[Bug Report] BoxChassis without collisions

### ISSUE THREAD

**Issue by dduro2020** [huawei-noah/SMARTS#2164]
### High Level Description

This chassis is defined in such a way that a cube of user-defined dimensions is created. The collisions of this object are created without assigning them to any collision group, so they do not work. On the other hand, AckermannChassis does have them correctly defined.

### Version

Latest.

### Steps to reproduce the bug

```shell
If you try to use the RLidar with vehicles using the BoxChassis, no obstacles are detected.
```

### System info

Ubuntu 22.04
Date: 09-2024

### Error logs and screenshots

I will share my graphical tests using pybullet inside SMARTS, the RLidar class and the different chassis:

BoxChassis before fix:
![Screenshot from 2024-11-07 02-18-34](https://github.com/user-attachments/assets/b87abf32-73cf-4467-a5e1-783e8b7e701c)

AckermanChassis (how it should look):
![Screenshot from 2024-11-07 02-19-10](https://github.com/user-attachments/assets/87bc3fc2-310e-411c-b37d-326ddd16d4e1)

### Impact (If known)

Basically you cannot detect obstacles because of this problem.

**Comment by dduro2020** [comment]
https://github.com/huawei-noah/SMARTS/pull/2165 Should close this issue.

**Comment by Gamenot** [comment]
#2165 has been merged.
